aesthetic_ants/penguin/enemy.py

```diff
@@ -0,0 +1,16 @@
+import pyglet
+import random
+
+
+class Enemy(pyglet.sprite.Sprite):
+
+    def __init__(self):
+        enemy_image = pyglet.resource.image('circle.png')
+        enemy_image.width = 50
+        enemy_image.height = 50
+        super().__init__(enemy_image)
+        self.velocity_x, self.velocity_y = random.randint(0, 100), random.randint(0, 100)
```

Comment:
This should be in separate lines.
```suggestion
        self.velocity_x = random.randint(0, 100)
        self.velocity_y = random.randint(0, 100)
```